tests/pql/test_query.py

```diff
@@ -195,7 +195,7 @@ def test_simply_with_left_query():
     assert result[0]["with"][0]["attr_right"] is None
 
 
-def test_simply_with_right_query():
+def test_simply_with_query():
```

Comment:
@Monikaa1909, @patrycjarecko: simply?

Reply:
z tego co widać to "simply" już było
zmienić to? i jak tak to co sugerujesz?

Reply:
nie wiem. Nie wiem co to jest simply.

Reply:
@SiwyKot41 o tym mowilam